### PR TITLE
ENH: Addition of ITKHistogramMatchingImage filter

### PR DESCRIPTION
**Pull request by fbudin69500**

### CODE REVIEW

File: Documentation/ITKImageProcessingFilters/ITKHistogramMatchingImage.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+ITKHistogramMatchingImage {#itkhistogrammatchingimage}
+=========================
+
+## Group (Subgroup) ##
+ITKImageProcessing (ITKImageProcessing)
```
Comment (thewtex) [inline Documentation/ITKImageProcessingFilters/ITKHistogramMatchingImage.md]: ImageProcessing (ITK IntensityTransformation)

File: ITKImageProcessingFilters/ITKHistogramMatchingImage.cpp [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+//
+// -----------------------------------------------------------------------------
+const QString ITKHistogramMatchingImage::getHumanLabel()
+{ return "ITK::Histogram Matching Image Filter (KW)"; }
+
+// -----------------------------------------------------------------------------
+//
+// -----------------------------------------------------------------------------
+const QString ITKHistogramMatchingImage::getSubGroupName()
+{ return "ITK ImageIntensity"; }
```
Comment (thewtex) [inline ITKImageProcessingFilters/ITKHistogramMatchingImage.cpp]: ITK IntensityTransformation

**Review by thewtex — CHANGES_REQUESTED** [review]
Minor tweaks suggested

**Comment by imikejackson** [comment]
+1

File: ITKImageProcessingFilters/ITKHistogramMatchingImage.cpp (hunk 2 above)
Comment (fbudin69500) [reply ITKImageProcessingFilters/ITKHistogramMatchingImage.cpp]: Addressed

**Pull request merged**